templates/python-beautifulsoup/requirements.txt

```diff
@@ -1,6 +1,7 @@
-# Add your dependencies here.
-# See https://pip.pypa.io/en/latest/reference/requirements-file-format/
-# for how to format them
+# Feel free to add your Python dependencies below. For formatting guidelines, see:
+# https://pip.pypa.io/en/latest/reference/requirements-file-format/
```

Comment:
How hard is it for a template user to transition to poetry, for instance? One day, I'd love to see a link to a guide on that here.

Reply:
We might consider offering two versions of the Python templates. One utilizes Poetry and the other uses the default pip with `requirements.txt`.

I would be concerned about updating templates to use Poetry only. Since it is a potential new entry barrier for newcomers unfamiliar with Poetry.

However, we could probably provide 2 versions of the templates, one with Poetry and one with the default pip and `requirements.txt`.

Also, we could integrate the Poetry into SDK & Client & Shared packages for better dependency management. This one would not create any new entry barrier.

I think we've already had some discussions with @fnesveda and @jirimoravcik regarding this topic. Tagging you guys in case you would like to say something about it. Thanks.

Reply:
We should check if there is demand for that (e.g. if someone did it by themselves). We have some analysis of what packages are users using in their Actors, we could check if someone has an Actor that uses Poetry.

Reply:
One day I'd love to see some customizable templates, with some wizard to create them, instead of several hard-coded templates.

You could choose:
1. Language (Python, TS, JS)
2. Language version (Python 3.8 - 3.12, Node 18 - 21, or maybe Bun)
3. Technology (Scrapy, Beautiful Soup, Crawlee, Cheerio...)
4. Package manager (NPM, Yarn, Pip, Poetry, ...)
5. Something else?

But that's a dream for the future, I can't imagine the template templating that this would require.